lib/build/builder.js

```diff
@@ -4,16 +4,14 @@ import {CompositeDisposable, Range} from 'atom'
 import fs from 'fs-extra'
 import path from 'path'
 import temp from 'temp'
+import {isValidEditor} from './../utils'
 
 class Builder {
-  constructor (goconfig) {
+  constructor (goconfig, linter) {
```

Comment:
Perhaps. This thing only has 3 functions so you could put a shim in there pretty easy.